pyscaffold/templates/skeleton.template

```diff
@@ -67,7 +67,7 @@ def parse_args(args):
 
 def main(args):
     args = parse_args(args)
-    print("The {}-th Fibonacci number is {}".format(args.n, fib(args.n)))
+    print("The {}-th Fibonacci number is {}".format(args.n, fib(int(args.n))))
```

Comment:
Instead of converting the parameter, couldn't we just add ``type=int`` to 

>      parser.add_argument(
         dest="n",
         help="n-th Fibonacci number",
         metavar="INT")